exercicios-para-casa/exercicios-taline/uriJudge/03-exercicio/script.js

```diff
@@ -0,0 +1,10 @@
+let x = Number(prompt('Digite um número inteiro:'))
```

Comment:
Esse exercicio conseguimos resolver com for ou while

let countPar = 0;
let countImpar = 0;
let countNegativo = 0;
let countPositivo = 0;


for(let i = 0; i < 5; i++){
  let number = Number(prompt('informe o valor ' + i))
  if(number % 2 === 0){
   ++countPar
  } else {
   ++countImpar
  }
  
  if(number < 0){
      ++countNegativo
  } else if (number > 0) {
     ++countPositivo
  }
}  
console.log('pares ', countPar)
console.log('impares ', countImpar)
console.log('positivos ', countPositivo)
console.log('negativos ', countNegativo)